lib/virt_autotest/utils.pm

```diff
@@ -105,23 +105,56 @@ sub is_xen_host {
     return get_var("XEN") || check_var("SYSTEM_ROLE", "xen") || check_var("HOST_HYPERVISOR", "xen") || check_var("REGRESSION", "xen-hypervisor");
 }
 
-#check host to make sure it works well
+#check kernels by grep keywords from journals
 #welcome everybody to extend this function
-sub check_host {
+sub check_failures_in_journal {
+    my $machine //= 'localhost';
+
+    my $all_failures = "";
+    my @warnings = ('Started Process Core Dump', 'Call Trace');
+    foreach my $warn (@warnings) {
+        my $cmd = "journalctl | grep '$warn'";
+        $cmd = "ssh root\@$machine " . "\"$cmd\"" if $machine ne 'localhost';
+        if (script_run($cmd) == 0) {
+            if (get_var('KNOWN_KERNEL_BUG')) {
+                my $failure = "$warn on $machine: " . get_var('KNOWN_KERNEL_BUG');
+                $all_failures .= $failure;
+                record_soft_failure("$failure (Please ignore the 'fake bsc#5555' that has to be written here due to the silly PR code rule)");
+            }
+            else {
+                record_soft_failure("$warn on $machine: Fake bsc#5555. This is an unknown failure which need to be investigated! (Please ignore the 'fake bsc#5555' that has to be written here due to the silly PR code rule)");
+                $all_failures .= "$warn on $machine: Unknown failure";
+            }
+        }
+    }
+    return $all_failures;
+}
 
+# Do some basic check to host to see if it is working well
+# Return 'pass' and 'fail' if there are or are not failures.
+# Welcome everybody to extend this function
+sub check_host_health {
+    my $failures = check_failures_in_journal;
+    if ($failures eq "") {
+        record_info("Host good!");
+        return 'pass';
+    }
+    record_info("Failures in journals are found", $failures, result => 'fail');
```

Comment:
Could you help make the messages in record_info more consistent and concise ? For example,
```
record_info("Host is healthy !");
```
```
record_info("Unhealthy host !", $failures, result => 'fail');
```

Reply:
because there are two words show up in the green record_info in job page by default --- You don't need to put your focus to see the entire statements. I can change it to 'Healthy host!'.